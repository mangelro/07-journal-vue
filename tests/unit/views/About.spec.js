import {shallowMount} from '@vue/test-utils'
import About from '@/views/About'

describe('Prueba de About View',()=>{

    // let wrapper
    // beforeEach(()=>
    // {
    // })

	test('Debe renderizar el componente correctamente contra el snapshot',()=>{
        const wrapper=shallowMount(About)
		expect(wrapper.html()).toMatchSnapshot()

	})


})