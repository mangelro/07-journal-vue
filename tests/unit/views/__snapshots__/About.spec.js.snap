// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de About View Debe renderizar el componente correctamente contra el snapshot 1`] = `<h2>About</h2>`;
